xsl/xsl-ns-stylesheets/xhtml/chunk-common.xsl

```diff
@@ -1209,7 +1209,7 @@ xmlns:exsl="http://exslt.org/common" xmlns:cf="http://docbook.sourceforge.net/xm
     <xsl:if test="//d:legalnotice                   and not($generate.legalnotice.link = 0)                   and not($html.head.legalnotice.link.types = '')">
       <xsl:call-template name="make.legalnotice.head.links"/>
     </xsl:if>
-
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/docsearch.js/1/docsearch.min.css" />
```

Comment:
This won't work on embedded versions of the documentation. Do we have an idea to work around this yet?